src/app_service/service/chat/service.nim

```diff
@@ -279,7 +279,8 @@ QtObject:
 
     for chat in self.channelGroups[communityId].chats:
       result.unviewedMentionsCount += chat.unviewedMentionsCount
-      if chat.muted:
+      # We count the unread messages if we are unmuted and it's not a mention, we want to show a badge on mentions
+      if chat.unviewedMentionsCount == 0 and chat.muted:
```

Comment:
Just to be sure, is it going to be reevaluated when the chat gets unmuted?

Reply:
No. Good point. That's always been an issue I guess.

With my fix, when you unmute, at least the channel will have a badge (before it wouldn't have), but the section will not.

I guess we can open an issue.

Reply:
Same for the category I guess? ;) That one is still not working either

Reply:
@caybro muted for category technically not exists :) muting category just a shortcut